packages/observable/src/observable.ts

```diff
@@ -10,63 +10,171 @@ import * as dependencyDetection from './dependencyDetection'
 import { deferUpdates } from './defer'
 import { subscribable, defaultEvent, LATEST_VALUE } from './subscribable'
 import { valuesArePrimitiveAndEqual } from './extenders'
+import type { Subscribable, MaybeSubscribable } from './subscribable'
 
-export function observable (initialValue) {
-  function Observable () {
+//#region Observable
+
+/**
+ * Represents a value that can be either a plain value or an observable.
+ */
+export type MaybeObservable<T = any> = T | Observable<T>;
+
+/**
+ * Defines the functions available on an observable.
+ */
+export interface ObservableFunctions<T = any> extends Subscribable<T> {
```

Comment:
not for now, but it occurred to me that this could be 

```ts
export interface ObservableFunctions<T = any> extends Subscribable<T>, Function {
```

Reply:
I hope `():T` is enough. Extending Function or CallableFunction requires callee, args and co. Had commented something similar on subscription.ts